Vienna/Sources/Main window/ArticleListView.m

```diff
@@ -1655,17 +1670,39 @@ -(NSArray *)markedArticleRange
 -(void)dealloc
 {
 	[[NSNotificationCenter defaultCenter] removeObserver:self];
+    NSUserDefaults *userDefaults = NSUserDefaults.standardUserDefaults;
+    [userDefaults removeObserver:self
+                      forKeyPath:MAPref_ShowEnclosureBar
+                         context:ObserverContext];
 	[splitView2 setDelegate:nil];
 	[articleList setDelegate:nil];
 }
 
 // MARK: Key-value observation
 
-
 - (void)observeValueForKeyPath:(NSString *)keyPath
                       ofObject:(id)object
                         change:(NSDictionary<NSKeyValueChangeKey,id> *)change
-                       context:(void *)context {
+                       context:(void *)context
+{
+    if (context != ObserverContext) {
+        [super observeValueForKeyPath:keyPath
+                             ofObject:object
+                               change:change
+                              context:context];
+        return;
+    }
+
+    if ([keyPath isEqualToString:MAPref_ShowEnclosureBar]) {
+        NSNumber *showEnclosureBar = change[NSKeyValueChangeNewKey];
+        if (showEnclosureBar.boolValue) {
```

Comment:
Cf. -refreshArticlePane : showing the enclosure view should be done if : 
- just one article is displayed
- the article has an enclosure